kernel-impl/src/main/resources/antisamy/high-security-policy.xml

```diff
@@ -191,6 +191,19 @@
             </regexp-list>
          </attribute>
 
+        <attribute name="data-src" description="One of the set of data-* attributes used to store custom data private to the page or application">
+            <regexp-list>
+                <regexp name="anything" />
```

Comment:
'anything' here refers to another rule (line 35) which says that attribute's value can match anything.

<regexp name="anything" value=".*" />

Some other ones specify a list of values 

                <literal value="screen"/>
                <literal value="tty"/>
                <literal value="tv"/>
                <literal value="projection"/>

or a regex.